### DISCUSSION TITLE
I live in a shit third world country!!

### DISCUSSION THREAD

**Discussion by peatu** [DIGITALCRIMINALS/OnlyFans#813] [Show and tell]
Greetings everyone i wanted to subscribe to a OF model but cant cz everytime i check out and almost finish the Id authentication doesnt work because it is not a supported international document(because my country is a nobody) hence i cant sub to any model because of that so do u have any suggestions? And thank you

**Comment by GreyBeardy** [comment]
use a vpn

**Comment by starboi4eva** [comment]
I live in one too. Passports should be supported.
